benchmarkcommon/src/test/java/com/ss/benchmark/httpclient/HCPerformanceTests.java

```diff
@@ -0,0 +1,395 @@
+package com.ss.benchmark.httpclient;
+
+import com.codahale.metrics.*;
+import com.ss.benchmark.httpclient.common.Payloads;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+/**
+ * @author sharath.srinivasa
+ */
+@Test(groups = "performance")
+public abstract class HCPerformanceTests {
+
+    protected static final String ECHO_DELAY_BASE_URL = "/echodelayserv/echo";
+    protected static final String ECHO_DELAY_SHORT_URL = "/echodelayserv/echo/short";
+    protected static final String ECHO_DELAY_LONG_URL = "/echodelayserv/echo/long";
+    protected static final String BASE_URL = "http://localhost:8080";
+
+    protected static final int EXECUTIONS = 10_000;
+
+
+    private static final Logger logger = LoggerFactory.getLogger(HCPerformanceTests.class);
+
+    protected final MetricRegistry metricRegistry = new MetricRegistry();
+    protected final ScheduledReporter reporter = ConsoleReporter.forRegistry(metricRegistry).convertDurationsTo(TimeUnit.MILLISECONDS).build();
+
+
+    private Set<CountDownLatch> latches = new HashSet<>();
+
+    private HC client;
+
+    @BeforeTest
+    public void initializeTest() {
+        reporter.start(1, TimeUnit.HOURS);
```

Comment:
I think this should be set to something like 30 seconds.  While your start-stop test log statements alleviates some of the need for this, a particular test might _appear_ to be hanging when it really isn't.  Seeing dropwizard stats every 30 seconds will help avoid that mistake.